src/main/java/seedu/planner/model/ModulePlanner.java

```diff
@@ -103,14 +103,18 @@ public void setUserProfile(UserProfile u) {
     }
 
     /**
-     * Returns all {@code Semester}s wrapped in an {@code ObservableList}.
+     * Returns a copy of all the {@code Semester}s.
      *
-     * @return An {@code ObservableList} containing all the {@code Semester}s
+     * @return A list of {@code Semester}s
      */
     @Override
-    public ObservableList<Semester> getSemesters() {
-        return FXCollections.unmodifiableObservableList(
-                FXCollections.observableList(semesters));
+    public List<Semester> getSemesters() {
```

Comment:
Why change it to `List<...>`?

Reply:
Cos got no need for semesters to be "observed"; they're not used in the UI directly. Only their underlying modules are used directly

Reply:
Okay